### ISSUE TITLE
Convert Error

### ISSUE THREAD

**Issue by zegufe2015** [daviddoria/Examples#5]
Hi David,

When I run stereo_calibrate.cxx I have an error like below. 
What is the problem?
What shoul I do?
Thank you.

../stereo_calibrate/main.cpp:94: error: cannot convert 'CvTermCriteria' to 'int' for argument '13' to 'double cv::stereoCalibrate(cv::InputArrayOfArrays, cv::InputArrayOfArrays, cv::InputArrayOfArrays, cv::InputOutputArray, cv::InputOutputArray, cv::InputOutputArray, cv::InputOutputArray, cv::Size, cv::OutputArray, cv::OutputArray, cv::OutputArray, cv::OutputArray, int, cv::TermCriteria)'
                     CV_CALIB_SAME_FOCAL_LENGTH | CV_CALIB_ZERO_TANGENT_DIST);
                                                                            ^

**Comment by daviddoria** [comment]
Are you talking about this file?
https://github.com/daviddoria/Examples/blob/master/c%2B%2B/OpenCV/StereoCalibration/StereoCalibration.cxx

(I'm not sure I'll know the answer anyway - it's been years since I've looked at this)

**Comment by zegufe2015** [comment]
Yes I am talking about https://github.com/daviddoria/Examples/blob/master/c%2B%2B/OpenCV/StereoCalibration/StereoCalibration.cxx

**Comment by daviddoria** [comment]
Perhaps the function signature changed? Which version of OpenCV are you using?

http://docs.opencv.org/3.0-beta/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html#double%20stereoCalibrate%28InputArrayOfArrays%20objectPoints,%20InputArrayOfArrays%20imagePoints1,%20InputArrayOfArrays%20imagePoints2,%20InputOutputArray%20cameraMatrix1,%20InputOutputArray%20distCoeffs1,%20InputOutputArray%20cameraMatrix2,%20InputOutputArray%20distCoeffs2,%20Size%20imageSize,%20OutputArray%20R,%20OutputArray%20T,%20OutputArray%20E,%20OutputArray%20F,%20int%20flags,TermCriteria%20criteria%29

vs

http://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html#stereocalibrate

If you figure it out, I'm happy to accept pull requests!
